cmds/cpud/serve.go

```diff
@@ -115,6 +114,12 @@ func serve() error {
 	if err != nil {
 		return err
 	}
+
+	if *mdnsEnabled {
+		host, _ := os.Hostname()
+		mdns_server, _ := server.Advertise(host, *port, *mdnsTxt)
```

Comment:
I understand that the intent here is to make this optional so the error is ignored. But future readers of the code might not understand this, so can you please add a comment about why the error is ignored? Thanks!